### PR TITLE
PE-28(testcases) and code refactoring

### PR DESCRIPTION
**Pull request by Ramakanth19**

### CODE REVIEW

File: Admin_Page.html [hunk 1]
```diff
@@ -65,7 +66,9 @@
 }
 </style>
 <body>
-
```
Comment (harshavardhanreddybindela) [inline Admin_Page.html:68]: Please remove this space

File: Admin_Page.html [hunk 2]
```diff
@@ -89,43 +92,40 @@
...
 </form>
+<!--add product form ended-->
+
+<!--delete a product form initiated-->
 <form method="POST" action="delete_product.php" style="border:1px solid #ccc">
   <div class="container">
     <p>Please fill the below details to delete aproduct.</p>
     <hr>
-
     <label for="ProductId"><b>Product Id</b></label>
```
Comment (harshavardhanreddybindela) [inline Admin_Page.html:102]: Please remove extra spacing in the code

File: app/Test_php.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    public function emptycart($con) {
+        $user="DELETE from cart_details";
+        $result=mysqli_query($con,$user);
+        $num1=1;
+        if($result)
+        {
+        return $num1;
+        }
+    }
+}
```
Comment (harshavardhanreddybindela) [inline app/Test_php.php:44]: The code is good

File: Admin_Page.html (hunk 1 above)
Comment (Ramakanth19) [reply Admin_Page.html:68]: done

File: Admin_Page.html (hunk 2 above)
Comment (Ramakanth19) [reply Admin_Page.html:102]: done

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9"
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\": "app"
+        }
+    }
+}
```
Comment (harshavardhanreddybindela) [inline composer.json:13]: please comment the code

File: test/SampleTest.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        $host = "localhost";
+        $usern = "root";
+        $pass = "";
+        $dbname = "pesticide";
+        $con = mysqli_connect($host, $usern, $pass, $dbname);
+        $result=$cal->emptycart($con);
+        $this->assertEquals(1, $result);
+    }
+
+}
```
Comment (harshavardhanreddybindela) [inline test/SampleTest.php:51]: The testcases are clear

**Pull request merged**